docs/modules/documentation/containers/search-input/search-input-docs.component.ts

```diff
@@ -16,5 +18,15 @@ export class SearchInputDocsComponent {
     searchInputDynamicHtml = searchInputDynHtml;
     searchInputDynamicTs = searchInputDynTs;
 
+    searchInputAsyncHtml = searchInputAsyncHtml;
+    searchInputAsyncTs = searchInputAsyncTs;
+
+    searchInputAsyncData = JSON.stringify([
+        {"text": "Apple", "alertMessage": "Apple Clicked"},
```

Comment:
single quote plus no need for quotes at all for the field name, no need to format it like real json if you're stringifying